README.md

```diff
@@ -8,6 +8,15 @@ If you don't have an account, you can create it here
 
 #### Notice: make sure that you don't push your SendGrid credentials in this repo. Otherwise, SendGrid can block your account
 
+## Google SMTP
```

Comment:
Change the description:
## Email senders
In this service we use several senders ("Google SMTP", "SendGrid") for sending emails to recipients. 

### Send Grid
You must have an account with API key that you will use in this app. If you don't have an account, you can create it here.
Notice: make sure that you don't push your SendGrid credentials in this repo. Otherwise, SendGrid can block your account

### Google SMTP
We use this sender in "Debug" and "Development" mode.
To use the service, you need to create a password for external applications on your account https://support.google.com/accounts/answer/185833?hl=en.
Also check this article to configure Google SMTP https://support.google.com/mail/answer/7126229?hl=en.

#### Notice: make sure that you don't push your Gmail SMTP credentials in this repo.